server/shared/storage/providers/amazon.js

```diff
@@ -22,17 +34,19 @@ class Amazon {
     config = validateConfig(config, schema);
 
     const s3Config = {
-      signatureVersion: 'v4',
-      accessKeyId: config.key,
-      secretAccessKey: config.secret,
+      credentials: {
+        accessKeyId: config.key,
+        secretAccessKey: config.secret
+      },
       region: config.region,
+      signatureVersion: 'v4',
```

Comment:
Move to top of an object, to reduce diff.

Reply:
Reverted code formatting